src/components/UserInfo.js

```diff
@@ -1,19 +1,24 @@
 export default class UserInfo {
-    constructor({ nameSelector, jobSelector }) {
-      this.nameElement = document.querySelector(nameSelector);
-      this.jobElement = document.querySelector(jobSelector);
-    }
-  
-    getUserInfo() {
-      return {
-        name: this.nameElement.textContent,
-        job: this.jobElement.textContent
-      };
-    }
-  
-    setUserInfo({ title, job }) {
-      this.nameElement.textContent = title;
-      this.jobElement.textContent = job;
-    }
+  constructor({ nameSelector, jobSelector, avatarSelector }) {
+    this._nameEl = document.querySelector(nameSelector);
+    this._jobEl = document.querySelector(jobSelector);
+    this._avatarElement = document.querySelector(avatarSelector);
   }
-  
\ No newline at end of file
+
+  getUserInfo() {
+    return {
+      name: this._nameEl.textContent,
+      job: this._jobEl.textContent,
+      avatar: this._avatarELement,
```

Comment:
There is no need to return the avatar here, since it's controlled by a different form. You can create a different getter for the avatar

Reply:
When do you mean about creating a different getter for the avatar?